airflow-core/docs/core-concepts/params.rst

```diff
@@ -409,3 +409,52 @@ Disabling Runtime Param Modification
 
 The ability to update params while triggering a Dag depends on the flag ``core.dag_run_conf_overrides_params``.
 Setting this config to ``False`` will effectively turn your default params into constants.
+
+To pre-populate values in the form when publishing a link to the trigger form you can call the trigger URL ``/dags/<dag_name>/trigger/single`` or ``/dags/<dag_name>/trigger/backfill``,
+and add query parameters to the URL.
+
+There are two trigger form URLs available, each supporting a different set of query parameters:
+
+* ``/trigger/single``:
+  - ``conf`` - JSON configuration.
+  - ``run_id`` - run identifier.
+  - ``logical_date`` - execution date in ``YYYY-MM-DDTHH:mm:ss.SSS`` format. Defaults to the current timestamp if not provided.
+  - ``note`` - note attached to the DAG run.
+
+* ``/trigger/backfill``:
+  - ``conf`` - JSON configuration, applied to all runs.
+  - ``from_date`` - start of the backfill window in ``YYYY-MM-DDTHH:mm:ss`` format.
+  - ``to_date`` - end of the backfill window in ``YYYY-MM-DDTHH:mm:ss`` format.
+  - ``max_active_runs`` - maximum concurrent runs. Defaults to ``1``.
+  - ``reprocess_behavior`` - determines how existing runs are reprocessed. Supported values are:
+
+    * ``failed`` - Missing and Errored Runs
+    * ``completed`` - All Runs
+    * ``none`` - Missing Runs
+
+  - ``run_backwards`` - if set to true, the backfill is scheduled in reverse order. Defaults to ``false``.
+
+The trigger form now supports two different ways of providing ``conf`` values. The available input methods are summarized in the table below:
```

Comment:
The term "now" is a bit relative... especially if you might read the same text in 10 years if the feature still exists.
```suggestion
The trigger form supports two different ways of providing ``conf`` values. The available input methods are summarized in the table below:
```